products/models.py

```diff
@@ -0,0 +1,129 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=30)
+
+    class Meta():
+        db_table = 'categories'
+
+
+class SubCategory(models.Model):
+    name = models.CharField(max_length=30)
+    category = models.ForeignKey('Category', on_delete=models.CASCADE)
+
+    class Meta():
+        db_table = 'sub_categories'
+
+
+class Product(models.Model):
+    name           = models.CharField(max_length=30)
+    price          = models.DecimalField(max_digits=8, decimal_places=2)
+    thunbnail_url  = models.CharField(max_length=2000)
+    is_new         = models.NullBooleanField()
+    is_sale        = models.NullBooleanField()
+    is_soldout     = models.NullBooleanField()
+    is_set         = models.NullBooleanField()
+    contents       = models.TextField() 
+    is_picked      = models.BooleanField(default=True, null=True)
+    created_at     = models.DateTimeField(auto_now_add=True)
+    sub_category   = models.ForeignKey('SubCategory', on_delete=models.CASCADE)
+    character      = models.ForeignKey('Character', on_delete=models.CASCADE)
+    discount_ratio = models.DecimalField(max_digits=8, decimal_places=2) 
+
+    class Meta():
+        db_table = 'products'
+
+
+class ProductProduct(models.Model):
+    product1 = models.ForeignKey('Product', on_delete=models.CASCADE)
+    product2 = models.ForeignKey('Product', on_delete=models.CASCADE)
+
+    class Meta():
+        db_table = 'products_products'
```

Comment:
네이밍에서 혼선이 있을 것 같아 제 생각을 추천 드립니다. 추천된 것을 기반으로 의견 추가하여 바꾸면 좋을 것 같습니다.
```suggestion
class ProductRelation(models.Model):
    reference = models.ForeignKey('Product', on_delete=models.CASCADE)
    related_product = models.ForeignKey('Product', on_delete=models.CASCADE)

    class Meta():
        db_table = 'product_relations'
```

Reply:
넵! 네이밍 변경하겠습니다~